src/BundleManager.js

```diff
@@ -175,19 +175,26 @@ class BundleManager {
     Logger.verbose(`\tLOAD: Attributes...`);
 
     const attributes = require(attributesFile);
-    let error = `\tAttributes file [${attributesFile}] from bundle [${bundle}]`;
     if (!Array.isArray(attributes)) {
+      let error = `\tAttributes file [${attributesFile}] from bundle [${bundle}]`;
```

Comment:
Since you moved error variable inside of if(..), use const
```suggestion
      const error = `\tAttributes file [${attributesFile}] from bundle [${bundle}]`;
```